crates/re_space_view_time_series/src/lib.rs

```diff
@@ -2,9 +2,16 @@
 //!
 //! A Space View that shows plots over Rerun timelines.
 
+mod aggregation;
+mod legacy_visualizer_system;
```

Comment:
I'm clearly missing a file or a rename, this file is not checked in yet!

Reply:
Oops, yeah. I named it back because it made the diff much easier to review but messed up the last pus.